package/tests/CMakeLists.txt

```diff
@@ -51,11 +51,11 @@ add_test(NAME basicIOmultithread COMMAND testIO.py --usleep 100 --io 10 --thread
 add_test(NAME basicIOmultiproc COMMAND testIO.py --usleep 100 --io 10  --procs 2)
 
 # Net Tests
-add_test(NAME basicNET COMMAND python testNET.py)
+add_test(NAME basicNET COMMAND python3 testNET.py)
 
 # Memory Tests
-add_test(NAME singleMem COMMAND python testMEM.py --procs 1)
-add_test(NAME childMem COMMAND python testMEM.py --procs 4)
+add_test(NAME singleMem COMMAND python3 testMEM.py --procs 1)
```

Comment:
I never really noticed this before. Is there a specific reason why some of our tests (e.g. this one) are `... COMMAND pythonX foo.py ...` and others (e.g. CPU ones) `... COMMAND foo.py ....`? Perhaps we can unify them.

Reply:
Interesting - I am really not sure why this is... see comment below